Examples/ExampleBlazor/Data/InfluxModel.cs

```diff
@@ -0,0 +1,696 @@
+using System.Diagnostics;
+using System.Globalization;
+using InfluxDB.Client.Api.Domain;
+using InfluxDB.Client.Core.Flux.Domain;
+using InfluxDB.Client.Writes;
+
+namespace ExampleBlazor.Data;
+
+public static class InfluxModel
+{
+    public static readonly Client Client = new()
+    {
+        Url = "http://localhost:8086",
+        Token = "my-token",
+        Org = "my-org"
+    };
+
+    public static async Task<bool> CheckClient(Client? client)
+    {
+        var influxDbClient = client?.GetClient();
+        return await influxDbClient?.PingAsync()!;
+    }
+
+    public static async Task<string> GetOrganizationId(Client client)
+    {
+        var influxDbClient = client.GetClient();
+        try
+        {
+            var orgList = await influxDbClient.GetOrganizationsApi().FindOrganizationsAsync(org: client.Org);
+
+            var tmp = orgList.First();
+            return (await influxDbClient.GetOrganizationsApi().FindOrganizationsAsync(org: client.Org))
+                .First().Id;
+        }
+        catch (Exception e)
+        {
+            Debug.WriteLine("Failed to load organization Id." + e);
+        }
+        finally
+        {
+            influxDbClient.Dispose();
+        }
```

Comment:
Please use `using` keyword instead explicitly disposing client. Use this behaviour on all places.

```suggestion
        using var influxDbClient = client.GetClient();
        try
        {
            var orgList = await influxDbClient.GetOrganizationsApi().FindOrganizationsAsync(org: client.Org);

            var tmp = orgList.First();
            return (await influxDbClient.GetOrganizationsApi().FindOrganizationsAsync(org: client.Org))
                .First().Id;
        }
        catch (Exception e)
        {
            Debug.WriteLine("Failed to load organization Id." + e);
        }
```